packages/api/package.json

```diff
@@ -41,6 +41,7 @@
         "ts-mockito": "^2.3.1",
         "typedoc": "^0.13.0",
         "typedoc-plugin-markdown": "^1.1.19",
-        "typescript": "^3.4.5"
+        "typescript": "^3.4.5",
+        "cennznut.js": "github:cennznet/cennznut.js#v1.0.0"
```

Comment:
it's fine for a POC, but I can't merge this before cennznut.js is published to npmjs.

p.s: you probably want to name it as under `@cennznet/` namespace

Reply:
the other way is, not touching deps, and replace codes of doughnut generation with a static doughnut.

Reply:
Good suggestions. Static way might be simplest option